lib/rules/jsx-sort-props.js

```diff
@@ -402,6 +441,9 @@ module.exports = {
     const reservedFirst = configuration.reservedFirst || false;
     const reservedFirstError = validateReservedFirstConfig(context, reservedFirst);
     const reservedList = Array.isArray(reservedFirst) ? reservedFirst : RESERVED_PROPS_LIST;
+    const customPropsFirst = configuration.customPropsFirst || false;
+    const customPropsFirstError = validateCustomPropsFirstConfig(context, customPropsFirst);
+    const customPropsList = Array.isArray(customPropsFirst) ? customPropsFirst : [];
```

Comment:
```suggestion
    const customPropsList = configuration.customPropsFirst;
```
no need to even ensure it's an array